lean/main/intro.lean

```diff
@@ -61,21 +61,21 @@ and Scala. In Coq, it's OCaml. In Agda, it's Haskell. In Lean 4, the meta code i
 mostly written in Lean itself, with a few components written in C++.
 
 One cool thing about Lean, though, is that it allows us to define custom syntax
-nodes and to implement our own meta-level routines to elaborate those in the
+nodes and implement meta-level routines to elaborate those in the
```

Comment:
"those" still is awkward sounding to my US English ears, rather than "them".

Reply:
```suggestion
nodes and implement meta-level routines to elaborate them in the
```